server/helpers/artistsFuncs.js

```diff
@@ -0,0 +1,101 @@
+const { setMaxListeners } = require('superagent');
+const artists = require('../data/artists');
+
```

Comment:
ngl I don't understand alot of the things done in this file so not sure if i can comment on it. But it looks impressive so there's that